README.md

```diff
@@ -48,49 +56,44 @@ Using Machine Insight the PLC data can be visualized and analyzed via a Gantt ch
 
 ### Used components
 
-- Industrial Edge Management (IEM) V 1.5.2-4 / V 1.11.8
-  - Databus Configurator V 2.0.0-5
-  - Common Connector Configurator V 1.8.1-4
-- Industrial Edge Device (IED) V 1.10.0.-9
-  - Databus V 2.1.0-4
-  - OPC UA Connector V 1.8.1-6
-  - Device Scanner Service V 1.6.0
-  - Machine Insight Conifgurator V 1.3.2
-  - Machine Insight V 1.3.2
-- Industrial Edge App Publisher V 1.10.5
-- Docker Engine V 20.10.10
-- Docker Compose V 1.28.5
-- TIA Portal V16
+- Industrial Edge Management (IEM) V 1.5.5-2 / V 1.16.11
+  - Databus Configurator V 3.0.0
+  - Common Connector Configurator V 1.10.1-2
+- Industrial Edge Device (IED) simatic-ipc-ied-os-2.1.0-22
+  - Databus V 3.0.0
+  - S7+ Connector V 1.4.0
+  - IIH Essentials V1.11.0
+  - Common Configurator V 1.11.1
+  - Registry Service V 1.11.0
+  - Machine Insight V 2.0.0
 - PLC: CPU 1515F-2
 
-The **Device Scanner Service** is optional but required for using the scan functionality in Machine Insight Configurator.
-
-The **OPC UA Connector** and the **Databus** are optional but required for using the machine status feature in Machine Insight Configurator.
```

Comment:
This reference somehow goes to an old version of the Readme.md file. 
The configurator chapter was already removed, since it is not necessary any more.